pyproject.toml

```
@@ -177,31 +177,29 @@ skip_install = true
 dependency_groups = ["lint"]
 commands = [["ruff", "check"]]
 
-[tool.tox.env."mypy"]
+[tool.tox.env."pyrefly"]
 runner = "uv-venv-lock-runner"
 skip_install = false
 dependency_groups = ["lint"]
-commands = [["mypy"]]
+commands = [["pyrefly", "check"]]
 
 [tool.uv]
 dev-dependencies = [
     "datamodel-code-generator>=0.26.2",
-    "mypy>=1.13.0",
+    "pyrefly>=0.13.0",
```

```suggestion
    "pyrefly>=0.13.0,<0.14.0",
```

To match dev environment spec and to prevent the almost guaranteed breakages in minor versions :smile: